src/Plugin/Condition/AdoType.php

```diff
@@ -114,8 +113,15 @@ public function evaluate() {
         }
       }
     }
+    else {
+      // This not an entity type that bears a strawberryfield. Therefore, this condition can not apply.
+      // However, it could still be negated and cause in a "TRUE" evaluation to flip and result in the "FALSE"
+      // being returned. Deal with that by removing the negation.
+      unset($this->configuration['negate']);
```

Comment:
Oh! Learned a thing here. So removing it means solves this because of
```
public function isNegated() {
    return !empty($this->configuration['negate']);
  }
```